packages/amplify-e2e-tests/src/__tests__/mismatched-parameters-fast-fail.test.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  initJSProjectWithProfile,
+  generateRandomShortId,
+  addFunction,
+  updateFunction,
+  amplifyPushExpectError,
+} from '@aws-amplify/amplify-e2e-core';
+import { addEnvironmentCarryOverEnvVars, checkoutEnvironment } from '../environment/env';
+
+describe('mismatched-parameters-fast-fail', () => {
```

Comment:
nit: describes + test names should combine together to create human-readable sentences describing what the test is proving.

Reply:
Ok I will update the names